src/pages/explore/CandidatesPage/components/CandidatesList.tsx

```diff
@@ -21,14 +24,49 @@ type VoteResult = {
   message: string
 }
 
+const AlreadyVotedIcon = () => (
+ <>
+    <img src={CheckAllIcon} className="me-2" />
+    <label style={{ color: '#6c757d' }}>Voted</label>
+  </>     
+)
+
 const CandidatesList = ({ api, activeAccount, candidates }: CandidatesListProps): JSX.Element => {
   const [showAlert, setShowAlert] = useState(false)
+  const [votes, setVotes] = useState<SocietyCandidate[]>([])
   const [voteResult, setVoteResult] = useState<VoteResult>({ success: false, message: '' })
+  const society = api?.query?.society
+  
   const showMessage = (result: VoteResult) => {
     setVoteResult(result)
     setShowAlert(true)
   }
 
+  const usePrevious = (value: any) => {
+    const ref = useRef()
+    useEffect(() => {
+      ref.current = value
+    })
+    return ref.current
+  }
+
+  const prevActiveAccount = usePrevious(activeAccount)
```

Comment:
Hey @danielSbastos, I didn't understand why this previous account logic was added. Can you explain?